### PR TITLE
remove node14 from build images

### PR DESCRIPTION
**Pull request by arroyc**

### CODE REVIEW

**Comment by arroyc** [comment]
There may be additional changes required where we have hardcoded node 14 in our tests etc .. so I triggered a nightly build to find that out ...

File: tests/Oryx.BuildImage.Tests/VersionInformationTest.cs [hunk 1]
```diff
@@ -226,12 +226,10 @@ public void Node_UsesLTSVersion_ByDefault_WhenNoExplicitVersionIsProvided(string
         [InlineData("8", "v" + NodeVersions.Node8Version)]
         [InlineData("10", "v" + NodeVersions.Node10Version)]
         [InlineData("12", "v" + NodeVersions.Node12Version)]
-        [InlineData("14", "v" + NodeVersions.Node14Version)]
```
Comment (kichalla) [inline tests/Oryx.BuildImage.Tests/VersionInformationTest.cs:229]: I am suspecting there are lot of other places where node 14 is testing

Comment (arroyc) [reply tests/Oryx.BuildImage.Tests/VersionInformationTest.cs:229]: yeah .. thats why I triggered a nightly build  to find out the failed tests .. will update this PR accordingly

**Comment by qianz2** [comment]
Could you also remove dotnet preview from build images

**Comment by arroyc** [comment]
> Could you also remove dotnet preview from build images

i  dint find it there ... seems like its removed already .. are you still seeing it ?

**Pull request merged**